cmd/go-tpc/tpch.go

```diff
@@ -26,19 +27,26 @@ var queryTuningVars = []struct {
 	{"tidb_prefer_broadcast_join_by_exchange_data_size", "ON"},
 }
 
-func appendQueryTuningVarsToConnParams() {
-	for _, v := range queryTuningVars {
-		if !strings.Contains(connParams, v.name) {
-			connParams = fmt.Sprintf("%s&%s=%s", connParams, v.name, v.value)
-		}
+// isSysVarSupported determines if a system variable is supported in given TiDB version
+// INFO: Should be optimized if some vars are changed in the future.
```

Comment:
```suggestion
// TODO: Every known sys var should have a minimal supported version and be checked individually. For now we just assume all sys vars are supported since 7.1.0.
```